packages/devextreme/testing/tests/DevExpress.ui.events/events.utils.tests.js

```diff
@@ -505,6 +506,55 @@ QUnit.module('event utils', () => {
     });
 });
 
+QUnit.module('getEventTarget module', () => {
+    test('getEventTarget returns the element on which the event was raised', function(assert) {
+        assert.expect(1);
+
+        const host = document.getElementById('qunit-fixture');
+        const div = document.createElement('div');
+
+        host.appendChild(div);
+
+        const customEvent = new Event('customEvent', {
+            bubbles: true,
+            composed: true,
+        });
+
+        host.addEventListener('customEvent', (event) => {
+            const target = getEventTarget(event);
+
+            assert.strictEqual(div, target, 'getEventTarget returned target correctly');
+        });
+
+        div.dispatchEvent(customEvent);
+    });
+
+    if(QUnit.urlParams['nojquery']) {
+        skipInShadowDomMode('getEventTarget returns the element on which the event was raised if there is ShadowDOM', function(assert) {
```

Comment:
So, we skip it because we emulate shadowDOM manually and we don't need one more level on wrapping to shadowDOM? Isn't it possible to use common wrapping and get rid of our manual wrapping?

Reply:
If we write a test through common wrapping mechanism we will also be able to remove `if(QUnit.urlParams['nojquery']) {`